solidity/ecdsa/deploy/91_authorize_in_random_beacon_chaosnet.ts

```diff
@@ -0,0 +1,27 @@
+import type { HardhatRuntimeEnvironment } from "hardhat/types"
+import type { DeployFunction } from "hardhat-deploy/types"
+
+const func: DeployFunction = async (hre: HardhatRuntimeEnvironment) => {
+  const { getNamedAccounts, deployments } = hre
+  const { deployer } = await getNamedAccounts()
+  const { execute } = deployments
+
+  const WalletRegistry = await deployments.get("WalletRegistry")
+
+  await execute(
+    "RandomBeaconChaosnet",
+    { from: deployer, log: true, waitConfirmations: 1 },
+    "setRequesterAuthorization",
+    WalletRegistry.address,
+    true
+  )
+}
+
+export default func
+
+func.tags = ["WalletRegistryAuthorizeInBeaconChaosnet"]
+func.dependencies = ["UpgradeRandomBeaconChaosnet", "WalletRegistry"]
+
+// Only execute for chaosnet deployments.
+func.skip = async (hre: HardhatRuntimeEnvironment): Promise<boolean> =>
+  !hre.network.tags.chaosnet
```

Comment:
Once we do it, let's transfer the ownership of `RandomBeaconChaosnet` to the `governance` (Threshold Council).

Reply:
Done in a7859770d0342956f7049a3a468fe74c181c3695.